htdocs/js/notebook/notebook_model.js

```diff
@@ -174,6 +174,35 @@ Notebook.create_model = function()
             }
             return changes;
         },
+        remove_selected_cells: function() {
+            var that = this;
+            _.chain(this.cells)
+            .filter(function(cell) {
+                return cell.is_selected();
+            })
+            .each(function(cell) {
+                that.remove_cell(cell);
+            });
```

Comment:
It doesn't really matter, but underscore isn't needed here - `this.cells.filter(...).forEach(...)` is fine.